src/index.js

```diff
@@ -101,6 +101,19 @@ const rule = (properties, options) =>
           }
         }
 
+        if (context.fix && !disableFix) {
+          // only if a fix code is available from config
+          if (options.autofixFunc[property + "-fix"]) {
+            //a is a local variable inside the fix function
+            var a = node.value;
+            // take the fix function from the config
+            node.value = eval(options.autofixFunc[property + "-fix"]);
```

Comment:
I would avoid eval and I would not provide a single function for each property, rather one function for all properties.
This function should receive the whole PostCSS API necessary for autofixing

Reply:
Regarding eval - I will proceed making the change. 

Reason for having a single function for each property - Lets say in the config we have properties such as `["margin", "fill"]`.
To have a auto fix, developer may have scss variables different for `margin` and different for `fill`. As example -So writing a different function seemed good. Or alternatively developer will have to include conditional statement in the fix function that will handle for the different properties. Looking for your suggestion here.


Regarding postCSS I would like to know why we will have to pass those to the function, because the fix function will change only a node's value and there is no need for traversing the CSS tree as the plugin only cares about the strict values and not about the descendants' or ancestors. 

Please let me know if my understanding is incorrect

Reply:
Update: resolved as per your suggestion. Removed eval and passing a function from config!